tests/packages/vso/prog/vso2.cpp

```diff
@@ -0,0 +1,26 @@
+// First test RE,
+// second WA,
+// other OK.
+// Group should have status RE.
```

Comment:
Could you also add tests where the final group status is different from the first test status? E.g. so that the tests are OK, OK, WA, OK, RE, OK, WA, and the final status should be RE.

Reply:
I added more tests 1f00fa795ab50f147d708732511a445b449f323c